src/providers/sh/util/scale/constants.js

```diff
@@ -1,11 +1,12 @@
 // @flow
-export type RegionToDC = {
+export type DC = {
```

Comment:
Shouldn't this be an enum?

Reply:
Probably but I'd rather get rid of it later.